sklearn/svm/_classes.py

```diff
@@ -407,7 +407,7 @@ class LinearSVR(RegressorMixin, LinearModel):
     Examples
     --------
     >>> from sklearn.svm import LinearSVR
-    >>> from sklearn.pipeline import make_pipeline
+    >>> from sklearn.pipeline import make_pipeline, Pipeline
```

Comment:
In this example, `make_pipeline` is called which returns a `Pipeline`, but importing itself `Pipeline` is not required.

For future reference, our CI runs doctests to make sure the examples can run. In other words, the CI will fail if an import was missing.

Reply:
Thank you for the clarification. I tried it standalone in a jupyter notebook and it did not work for me.

Reply:
For copy and pasting into a jupyter notebook, it's best to hover over the top right and hide the output:

![Screen Shot 2022-04-21 at 12 47 21 PM](https://user-images.githubusercontent.com/5402633/164510857-437ac690-9918-4fdf-b7cc-725f53c87eac.jpg)